src/People.tsx

```diff
@@ -0,0 +1,113 @@
+import { Loader } from './components/Loader';
+import { getPeople } from './api';
+import { useEffect, useState } from 'react';
+import { Person } from './types';
+import { PersonLink } from './components/PersonLink';
+import { useParams } from 'react-router-dom';
+
+export const People = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(false);
```

Comment:
The `errorMessage` state is initialized as a boolean but is named in a way that suggests it might hold a string message. Consider renaming it to `hasError` for clarity, or change its type to hold an error message string if that's the intended use.